test/integration/rotation/05-check-svids

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# 15 checks should be enough for the server to prepare and rotate into a new CA
```

Comment:
comment above is phrased in seconds, this one in "checks" where the checks have a 3 second sleep. Lets keep this consistent.

Reply:
Rewording...